.dockerignore

```diff
@@ -1,6 +1,9 @@
 # Version Control
 .git
 
+**settings_custom.py
```

Comment:
Could you include a `settings_custom.example.py` file for reference? It would better help me understand what should exist in this file

Reply:
Yup, will do

Reply:
done